### PR TITLE
RNAsum v0.4.7

### PR DESCRIPTION
**Pull request by skanwal**
This release resolves two issues:

1. Handle missing fusion calls

- Related issue https://github.com/umccr/RNAsum/issues/90
- Related PR https://github.com/umccr/RNAsum/pull/94

2. Chrome bug 

- Related issue https://github.com/umccr/RNAsum/issues/93
- Related PR https://github.com/umccr/RNAsum/pull/96

Tested in dev with `wfr.6327837a1f2a4cd485375ad396f2bc69`.

### CODE REVIEW

**Comment by victorskl** [comment]
[Portal Release - Snowshoe: RNAsum Handle missing fusion calls](https://trello.com/c/OWGezQKg/1464-portal-release-snowshoe-rnasum-handle-missing-fusion-calls)

**Comment by victorskl** [comment]
[Portal Release - Snowshoe: RNAsum Chrome bug fix](https://trello.com/c/8C25jzLj/1458-portal-release-snowshoe-rnasum-chrome-bug-fix)

File: tools/rnasum/0.4.7/rnasum__0.4.7.cwl [hunk 1]
```diff
@@ -0,0 +1,307 @@
+cwlVersion: v1.1
+class: CommandLineTool
+
+# Extensions
+$namespaces:
+    s: https://schema.org/
+    ilmn-tes: http://platform.illumina.com/rdf/ica/
```
Comment (alexiswl) [inline tools/rnasum/0.4.7/rnasum__0.4.7.cwl:7]: Should be `ilmn-tes: https://platform.illumina.com/rdf/ica/`

Comment (skanwal) [reply tools/rnasum/0.4.7/rnasum__0.4.7.cwl]: updated in [74f092d](https://github.com/umccr/cwl-ica/pull/350/commits/74f092dd51cb782f538c1ca240df2ab8c4c6354e)

**Review by alexiswl — APPROVED** [review]

**Pull request merged**